static/docs/commands-reference/move.md

```diff
@@ -59,16 +59,16 @@ name:
 
 ```dvc
 $ dvc move data.csv other.csv
-$ cat data.csv.dvc
+$ cat other.csv.dvc
 ```
 
-And here is the updated content of the `data.csv.dvc`:
+And here is the updated content of the `other.csv.dvc`:
 
 ```yaml
-md5: 3d1a3e5a5b662490e198d6a6ae84984b
```

Comment:
don't see any strong reason to update yaml files here

Reply:
the md5 values for both yaml blocks will be same.

Reply:
k, if you got those new values by running the commands .. it makes sense to update md5 of the files